### ISSUE TITLE
Resolve the issue between rmarkdown and badges

### ISSUE THREAD

**Issue by choisy** [r-and-gama/rama#35]

**Comment by choisy** [comment]
Below is the error I get when I `knit()` the `README.Rmd` file:
``` r
pandoc: Could not fetch http://www.r-pkg.org/badges/version/rama
TlsExceptionHostPort (HandshakeFailed (Error_Misc "user error (unexpected type received. expecting handshake and got: Alert [(AlertLevel_Fatal,HandshakeFailure)])")) "www.r-pkg.org" 80
Erreur : pandoc document conversion failed with error 67
Exécution arrêtée
```
Any idea?

**Comment by LucieContamin** [comment]
It may be because of an old version of pandoc. Can you try to upgrade it? I can give you two links for downloading the last version: 
https://pandoc.org/installing.html
http://www.texts.io/support/0003/

**Comment by choisy** [comment]
Works perfectly well! Thanks. Fixed in commit 8c647ed6c2db7660bf732c6fe43edcb8a6ed4ed5
